website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,111 @@
+
+### Collaborate with Others
+Open source is all about collaboration. Don't be afraid to reach out to other contributors for help or feedback. You can also collaborate with other contributors by reviewing their code, providing feedback, and testing their changes. This will help you build relationships with other contributors and improve your coding or other skills you bring to the community. Contributing to an open source project doesn't require permission. Once you've done sufficient research and have a clear understanding of where your skills can help the community, don't hesitate to dive in and start contributing.
+
+### Asking for help
+
+Getting help from other contributors is a critical aspect of contributing to open source projects, but it's essential to know how much help is appropriate to ask for. It's okay to ask for assistance if you've made a genuine effort to learn the material beforehand, such as reading the documentation or using the software. Try to limit your requests to project-specific issues and avoid seeking help for general mentoring in open source or programming languages. 
+
+While seeking guidance on pull requests can be helpful, it's essential to keep in mind that simply asking for someone to provide you with the exact code or wording to use may not be the most effective approach. Instead, put in effort to create your own content and seek feedback on how you can improve your contribution. This way, you can work collaboratively with other contributors to achieve the project's objectives and build strong relationships within the community. 
+
+Remember, the goal is to help the project and maintainers, so ensure that your requests contribute to their efforts and do not increase their workload. By following these guidelines, you can build strong relationships with other contributors while improving your coding skills.
+
+### Technical Contributors
+If you're a technical contributor, there are some specific tools and technologies that you should be familiar with. Here are some of the key skills you'll need to contribute to open source:
+
+**Programming Languages**: Most open source projects are written in one or more programming languages. You'll need to be proficient in at least one language to contribute to a project. The specific language will depend on the project you choose.
+
+**Testing**: Testing is a critical part of software development. You'll need to be familiar with testing frameworks and tools to contribute to open source. This includes both unit testing and integration testing.
+
+### Non-Code Contributors
+Even if you're not a technical contributor, there are still many ways you can contribute to open source. Here are some of the key skills you'll need:
+
+**Documentation:** Most open source projects have documentation that needs to be written and maintained. If you're a good writer, you can contribute by writing documentation, tutorials or blog posts.
```

Comment:
Not just if they're already good, contributing can be a way to get good :)
```suggestion
**Documentation:** Most open source projects have documentation that needs to be written and maintained. If you're interested in writing, you can contribute by writing documentation, tutorials, or blog posts.
```

Reply:
Fixed, thanks